.github/workflows/e2e.yml

```diff
@@ -135,6 +135,7 @@ jobs:
         uses: ./.github/actions/build-mage
       - name: Install Node and Dependencies
         uses: ./.github/actions/install-node-and-deps
+      - uses: browser-actions/setup-chrome@stable
```

Comment:
For reference: This is required because the end to end tests run on multiple machines, and the expectation is that all of these machines run the same Chrome version. Apparently GitHub runners at the time of writing are split between Chrome 101 and Chrome 102, so in order to avoid flakes we need to make sure that Chrome is updated.

Even this PR had this particular flake - see the run at https://github.com/TheThingsNetwork/lorawan-stack/actions/runs/2422495024 .